rules/windows/defense_evasion_solarwinds_backdoor_service_disabled_via_registry.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/12/14"
+maturity = "production"
+updated_date = "2020/12/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a SolarWinds binary modifying the start type of a service to be disabled. An adversary may abuse this
+technique to manipulate relevant security services.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "SolarWinds Process Disabling Services via Registry"
+references = [
+    "https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html",
+]
+risk_score = 47
+rule_id = "b9960fef-82c6-4816-befa-44745030e917"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+registry where registry.path : "HKLM\\SYSTEM\\ControlSet*\\Services\\*\\Start" and registry.data.strings == "4" and
```

Comment:
when i tested with winlogbeat it came across like `HKLM\System\CurrentControlSet\Services\*\Start`

should we add a wildcard to the front of controlset to cover all bases `"HKLM\\SYSTEM\\*ControlSet*\\Services\\*\\Start"`

Reply:
good one, endpoint captures all with same path, guess this is something to note somewhere (it will apply to all related use cases, IIRC sysmon and endpoint both logs \ControlSet* and not the registry symlink \CurrentControlSet*)